### ISSUE TITLE
Fails on < inside a <pre> tag

### ISSUE THREAD

**Issue by jerel** [tildeio/simple-html-tokenizer#20]
In an Ember template if I display a snippet of code it fails thinking that it's the opening tag of an element. Example:

```
<pre>
    for (var i=0; i < 10; i++) {
        console.log(i);
    }
</pre>
```

Fails with `Unclosed element`i;`(on line 2).` Naturally I don't want an `&lt;` displayed in that code snippet.

**Comment by mmun** [comment]
Looks like we're missing the "anything else" parser error for the tag open state. See http://www.w3.org/TR/html5/syntax.html#tag-open-state.

FYI this is a parse error and you should be careful to rely on it. For example

```
<pre>
    for (var i=0; i<n; i++) {
        console.log(i);
    }
</pre>
```

gets parsed in a strange way.

**Comment by Turbo87** [comment]
tl;dr `<` in `<pre>` still needs to be escaped. seems like this issue can be closed.

**Comment by mmun** [comment]
Would be interesting to explore a lint specialized to `<` inside of `<pre>`.

**Comment by mmun** [comment]
The error in the original report says:

```
Unclosed element`i;`(on line 2).
```

This could be improved as well.
